deploy/cluster_role_binding.yaml

```diff
@@ -1,11 +1,14 @@
-kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
 metadata:
   name: hyperconverged-cluster-operator
+  labels:
+    operator.hco.kubevirt.io: ""
 subjects:
 - kind: ServiceAccount
   name: hyperconverged-cluster-operator
+  namespace: kubevirt
 roleRef:
-  kind: Role
-  name: hyperconverged-cluster-operator
-  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
```

Comment:
We need more granular rules than "cluster-admin" if we want to properly build a CSV for OLM.

Reply:
Right, I kept the role around for that reason.